ddtrace/contrib/dbapi/__init__.py

```diff
@@ -75,6 +75,15 @@ def callproc(self, proc, args):
             finally:
                 s.set_metric("db.rowcount", self.rowcount)
 
+    def __enter__(self):
```

Comment:
we're having issues with python 3+:
```
======================================================================
ERROR: tests.contrib.psycopg.test_psycopg.TestPsycopgPatch.test_cursor_ctx_manager
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/dd-trace-py/.tox/py34-contrib/lib/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/ubuntu/dd-trace-py/tests/contrib/psycopg/test_psycopg.py", line 94, in test_cursor_ctx_manager
    if version[0] == 2 and version[1] < 5:
TypeError: 'map' object is not subscriptable
```

We need to fix the condition to skip the test before merging.